### PR TITLE
Fix SIMVA Front

### PR DESCRIPTION
**Pull request by jujusb**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ let default_protocol_ports = {
 
 config.simva = {}
 config.simva.port  = parseInt(process.env.SIMVA_PORT || 3050);
-config.simva.host = process.env.SIMVA_HOST || 'simva.external.test'
+config.simva.host = process.env.SIMVA_HOST || 'external.test'
```
Comment (imartinezortiz) [inline config.js:10]: I think the previous default value is ok

File: public/activities/miniopainter.js
Comment (imartinezortiz) [inline public/activities/miniopainter.js:1]: Not specific to this change, but related to new code that requires to generate a string. Note that current browsers supports the backticks string definition that is far easier to use in a similar problem like this one.

File: routes/routes/users.js
Comment (imartinezortiz) [inline routes/routes/users.js:1]: Would be nice to migrate in the future to a Backend For Fronted approach, so the JS of the frontend do not require to deal with JWT, just using "credentials" (e.g. cookies) once the user has logged in. This is the common approach to SPA nowadays

**Review by imartinezortiz — CHANGES_REQUESTED** [review]

File: routes/routes/users.js
Comment (jujusb) [reply routes/routes/users.js:1]: issue opened

File: public/activities/miniopainter.js
Comment (jujusb) [reply public/activities/miniopainter.js:1]: fixed in all scripts

**Pull request merged**